src/routes/timedOrders.ts

```diff
@@ -119,6 +120,7 @@ export const createTimedOrder = async (req, res) => {
   const person = await getPerson(personId);
 
   const {
+    change_request_enabled: changeRequestEnabled,
```

Comment:
I remember that it was present on sepa transfer only. TO and SO were throwing an error from solaris